app/views/lines/_tiles.html.erb

```diff
@@ -1,7 +1,7 @@
 <% lines.uniq.each do |line| %>
 	<% unless defined? current_line and line == current_line %>
 		<div class="line-tile">
-			<%= link_to line_path(line) do %>
+			<%= link_to line_path(line.onestop_id) do %>
```

Comment:
I don't think this change was needed. We do `line_path(line)` elsewhere, and it works fine. Take a look at `lines/_show.html.erb`. However, I am having trouble testing this specifically so if you can show me where it was broken, I might be wrong.